.github/workflows/main.yml

```diff
@@ -14,6 +14,7 @@ jobs:
         node-version: [16.x]
     env:
       ENVIRONMENT: ${{ vars.ENVIRONMENT }}
+      AWS_ACCOUNT_ID: ${{ vars.AWS_ACCOUNT_ID }}
```

Comment:
Is it ok to have this as `vars` instead of `secrets`?

Reply:
I asked it to @w3dani and he said it was ok